tests/test_sanity.py

```diff
@@ -491,14 +489,10 @@ def test_emcc_multiprocess_cache_access(self):
     # Exactly one child process should have triggered libc build!
     self.assertEqual(num_times_libc_was_built, 1)
 
-  @parameterized({
-    '': [False],
-    'response_files': [True]
-  })
-  def test_emcc_cache_flag(self, use_response_files):
+  def test_emcc_cache_flag(self):
```

Comment:
why remove the testing of response files?

Reply:
You are right, maybe I can keep this?   this is part of the revert of #13954, but maybe this new test would have worked prior too.

Reply:
lgtm with the tests not removed (they can help in the future perhaps). but if they failed before then maybe not worth investigating in depth.

Reply:
I was able to keep on the of the tests (--cache via response file) but setting `--em-config` via response file doesn't work after this change (but it also has never worked until #13954 which landed yesterday).

The point of this change is to get this working and then revisit how we might support `--em-config` in a response files.